frontend/packages/dev-console/integration-tests/features/pipelines/pipeline-Display_task_runs_ODC-3712.feature

```diff
@@ -0,0 +1,40 @@
+Feature: Display task runs page
+    As a developer, I would like to navigate from a PipelineRun to the related TaskRuns and their associated pods so that I can debug my pipelines by looking at the volumes, secrets, etc that are made available to the taskruns and pods.
+
+Background:
+    Given user is at developer perspective
+    And user has installed OpenShift Pipelines Operator
+    And user has selected "aut-pipeline-taskruns" namespace
+
+
+@regression
+Scenario: Task runs tab
+    Given user is at pipeline details page
+    And pipeline runs are present
+    When user clicks on pipeline runs tab
+    And user clicks on a pipeline run to see Task Runs tab
+    And user clicks on Task Runs tab
+    Then user can see different task runs based on number of tasks present in pipeline
+    And user can see "Name", "Task", "Pod", "Status" and "Started" columns
+
+
+@regression
+Scenario: Options in breadcrumbs of task runs
+    Given user has pipeline runs present
+    And user is on Pipeline Runs Details page
+    When user clicks on Task Runs tab
+    And user clicks on breadcrumb of a task run
+    Then user can see "Edit labels", "Edit annotations", "Edit Task Run" and "Delete Task Run"
+
+
+@regression
+Scenario: Task Runs Details page
+    Given user is at pipeline details page
+    And pipeline runs are present
+    When user clicks on pipeline runs tab
+    And user clicks on a pipeline run
+    And user clicks on Task Runs tab
+    And user clicks on a task run
+    Then user is redirected to Task Run Details tab
+    And user can see "Details", "Log", "YAML" and "Events" tab
+    And user can see Status and Log snippet in "Details" tab
```

Comment:
Break into two scenarios

1. For a succeeded task run only `status` and `pod` fields are shown
![image](https://user-images.githubusercontent.com/9964343/100889619-5990a300-34dd-11eb-95ec-d5215cf7d7a8.png)
2. For a failed task run, following fields are shown
![image](https://user-images.githubusercontent.com/9964343/100889672-6b724600-34dd-11eb-843f-d618eae512d1.png)

Reply:
Added two scenarios for passed and failed task runs